srcpkgs/WindowMaker/template

```diff
@@ -19,9 +19,10 @@ conf_files="
 short_desc="X11 window manager with a NEXTSTEP look and feel"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later, WTFPL"
-homepage="http://www.windowmaker.org/"
-distfiles="http://windowmaker.org/pub/source/release/${pkgname}-${version}.tar.gz"
-checksum=f22358ff60301670e1e2b502faad0f2da7ff8976632d538f95fe4638e9c6b714
+homepage="https://www.windowmaker.org/"
+changelog="http://www.windowmaker.org/news/"
+distfiles="https://github.com/window-maker/wmaker/releases/download/wmaker-${version}/WindowMaker-${version}.tar.gz"
+checksum=4fe130ba23cf4aa21c156ec8f01f748df537d0604ec06c6bbcec896df1926f6d
```

Comment:
```suggestion
checksum=4fe130ba23cf4aa21c156ec8f01f748df537d0604ec06c6bbcec896df1926f6d
# tests various things against documentation, doesn't make sense to run
make_check=no
```
they're doing weird things like checking if the configure script has different weird arguments, and it's been happening since at least the previous version, so just disable them imo

Reply:
Ok will do. I will make that change and if the checks pass I will try to do a commit with all the changes in a single commit. Thanks!